docs/sources/configure-client/_index.md

```diff
@@ -60,9 +60,9 @@ To get started choose one of the integrations below:
           <a href="https://grafana.com/docs/pyroscope/next/configure-client/language-sdks/java/">Documentation</a><br />
           <a href="https://github.com/grafana/pyroscope/tree/main/examples/java-jfr/rideshare" title="java-examples">Examples</a>
       </td>
-      <td align="center"><a href="https://grafana.com/docs/pyroscope/next/configure-client/language-sdks/ebpf/"><img src="https://user-images.githubusercontent.com/23323466/178160548-e974c080-808d-4c5d-be9b-c983a319b037.png" width="100px;" alt=""/><br />
+      <td align="center"><a href="https://grafana.com/docs/pyroscope/next/configure-client/grafana-agent/ebpf"><img src="https://user-images.githubusercontent.com/23323466/178160548-e974c080-808d-4c5d-be9b-c983a319b037.png" width="100px;" alt=""/><br />
         <b>eBPF</b></a><br />
-          <a href="https://grafana.com/docs/pyroscope/next/configure-client/language-sdks/ebpf/" title="Documentation">Documentation</a><br />
+          <a href="https://grafana.com/docs/pyroscope/next/configure-client/grafana-agent/ebpf" title="Documentation">Documentation</a><br />
           <a href="https://github.com/grafana/pyroscope/tree/main/examples/ebpf" title="examples">Examples</a>
```

Comment:
hey @korniltsev can you update our eBPF example to match the new docs? 
https://github.com/grafana/pyroscope/tree/main/examples/ebpf